### ISSUE TITLE
Range Error: Invalid array length

### ISSUE THREAD

**Issue by SteveMcArthur** [bevry/progressbar#4]
The latest version of progressbar seems to have introduced an `Invalid array length` error. When running DocPad it falls over. However, downgrading progressbar to 1.0.3 seems to solve the problem

```
info: DocPad listening to http://127.0.0.1:9778/ on directory F:\temp\out
info: LiveReload listening to new socket on channel /docpad-livereload
Performing generate tasks ➞  Reset our collections at 0/1 0% -------------------RangeError: Invalid array length
  at ProgressBar.render (F:\temp\node_modules\progress\lib\node-progress.js:131:14)
  at ProgressBar.tick (F:\temp\node_modules\progress\lib\node-progress.js:86:8)
  at ProgressBar.<anonymous> (F:\temp\node_modules\progressbar\es2015\index.js:87:26)
  at emitOne (events.js:77:13)
  at ProgressBar.emit (events.js:169:7)
  at ProgressBar.setTick (F:\temp\node_modules\progressbar\es2015\index.js:178:9)
  at _Class.<anonymous> (F:\temp\node_modules\docpad\out\lib\docpad.js:1019:62)
  at emitOne (events.js:77:13)
  at _Class.emit (events.js:169:7)
  at _Class.TaskGroup.addItem (F:\temp\node_modules\taskgroup\out\lib\taskgroup.js:762:12)
  at _Class.TaskGroup.addTask (F:\temp\node_modules\taskgroup\out\lib\taskgroup.js:809:19)
  at DocPad.generate (F:\temp\node_modules\docpad\out\lib\docpad.js:3286:7)
  at Task.<anonymous> (F:\temp\node_modules\bal-util\out\lib\flow.js:42:21)
  at ambi (F:\temp\node_modules\ambi\es5\lib\ambi.js:98:14)
  at Domain.fireMethod (F:\temp\node_modules\taskgroup\out\lib\taskgroup.js:397:23)
  at Domain.run (domain.js:228:14)
  at Task.fire (F:\temp\node_modules\taskgroup\out\lib\taskgroup.js:435:27)
  at Immediate._onImmediate (F:\temp\node_modules\taskgroup\out\lib\taskgroup.js:452:26)
  at processImmediate [as _immediateCallback] (timers.js:383:17)
```

Tested this on a newly downloaded skeleton and same problem. I'm running on windows 10. Tested on DocPad  6.74 to 6.78.1

**Comment by balupton** [comment]
Could be a regression. Would you be able to look into this? Will be back tomorrow.

**Comment by balupton** [comment]
Should be fixed in v1.1.1.
